x-pack/plugins/uptime/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "./target/types",
+    "emitDeclarationOnly": true,
+    "declaration": true,
+    "declarationMap": true
+  },
+  "include": [
+    "common/**/*",
+    "public/**/*",
+    "server/**/*",
+    "../../../typings/**/*",
+    "public/**/*.json",
```

Comment:
The glob syntax above appeared to ignore JSON files.

Reply:
since there are just two json files, maybe it doesn't make sense to use wildcard pattern